gui/src/components/export/export-result-button.tsx

```diff
@@ -0,0 +1,52 @@
+import { Button } from "../ui/button";
+import { Popover, PopoverContent, PopoverTrigger } from "../ui/popover";
+import {
+  Select,
+  SelectContent,
+  SelectItem,
+  SelectTrigger,
+  SelectValue,
+} from "../ui/select";
+import OptimizeTableState from "../table-optimized/OptimizeTableState";
+import { useCallback } from "react";
+
+export default function ExportResultButton({
+  data,
+}: {
+  data: OptimizeTableState;
+}) {
+  const onExportClicked = useCallback(() => {
+    // do something here
```

Comment:
Please add implementation here

Reply:
thank i'll work on it when i got the time